File: launchdarkly/ruby/src/PatchOAuthClientExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

patch_operation_1 = LaunchDarklyClient::PatchOperation.new
patch_operation_1.op = "replace"
patch_operation_1.path = "/name"

patch_operation = [
    patch_operation_1,
]

begin
    response = LaunchDarklyClient::OAuth2ClientsApi.new.patch_o_auth_client(
        "clientId_string", # client_id
        patch_operation,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling OAuth2ClientsApi#patch_o_auth_client: #{e}"
end

File: launchdarkly/python/src/PatchOAuthClientExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    patch_operation_1 = models.PatchOperation(
        op="replace",
        path="/name",
    )

    patch_operation = [
        patch_operation_1,
    ]

    try:
        response = api.OAuth2ClientsApi(api_client).patch_o_auth_client(
            client_id="clientId_string",
            patch_operation=patch_operation,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling OAuth2ClientsApi#patch_o_auth_client: %s\n" % e)
